.c3i/authorized_users.yml

```
@@ -1256,3 +1256,8 @@ authorized_users:
 - piliugin-anton
 - Drllap
 - i-curve
+- Mi-La
+- gureedo
+- cbaecker
+- ErixenCruz
```

```suggestion
- ErixenCruz
- OrianeGourdyStilla
```

Asked to be included last week with others newcomers and has a PR, but the CI bot didn't include. Probably some restriction due account age.